apps/docs/src/app/platform/component-docs/platform-forms/platform-date-picker/platform-date-picker-docs.component.html

```diff
@@ -0,0 +1,62 @@
+<fd-docs-section-title [id]="'def'" [componentName]="'DatePicker'">
```

Comment:
when we're doing bindings with just strings we don't need `[id]="'mystring'"` we can just use `id="mystring"` Same is true for `[componentName]`

Reply:
Done. have made the correction in all files.